docs/modularization.rst

```diff
@@ -28,8 +29,9 @@ To load schema from file or directory, you can use the ``load_schema_from_path``
     # build an executable schema
     schema = make_executable_schema(type_defs)
 
-    # Start server that can't execute any queries, but allows you to browse your schema
-    start_simple_server(schema)
+    app = GraphQL(schema)
```

Comment:
Move this below next two comments and reword those to something like "Use schema to construct GraphQL app for an ASGI server"?